src/plugins/VideoRoomPlugin.ts

```diff
@@ -367,11 +396,17 @@ export class VideoRoomPlugin extends BasePlugin {
     const jsepOffer = await this.rtcConnection.createOffer(this.offerOptions);
     await this.rtcConnection.setLocalDescription(jsepOffer);
 
-    const confResult = await this.sendMessage({
-      request: 'configure',
-      ...options,
-    }, jsepOffer);
+    const confResult = await retryPromise(
+      () => this.sendMessage({
```

Comment:
Would be nice to save this parameter in a variable above

Reply:
Done. Hopefully I got you correctly